Discovery/Discovery/Presentation/NativeDiscovery/CourseDetailsView.swift

```diff
@@ -177,10 +177,12 @@ public struct CourseDetailsView: View {
             }
             
             // MARK: - Offline mode SnackBar
-            OfflineSnackBarView(connectivity: viewModel.connectivity,
-                                reloadAction: {
-                await viewModel.getCourseDetail(courseID: courseID, withProgress: false)
-            })
+            if viewModel.courseState() != .enrollOpen {
+                OfflineSnackBarView(connectivity: viewModel.connectivity,
+                                    reloadAction: {
+                    await viewModel.getCourseDetail(courseID: courseID, withProgress: false)
+                })
+            }
```

Comment:
**Question:** Shouldn't the `OfflineSnackBarView` only appear if there's no internet connection? The condition seems to be checking something different.

Reply:
This was done to avoid duplicating the message about the lack of Internet in this case:
![Simulator Screenshot - iPhone 15 Pro Max - 2024-03-20 at 15 22 11](https://github.com/openedx/openedx-app-ios/assets/128456094/5976049d-7e9a-4c75-855e-9e4d2e79ef15)